src/client/components/vision2/install.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { ComponentType } from 'react'
+
+import { NavigationConfiguration } from '../../navigation'
+import { NUsightNetwork } from '../../network/nusight_network'
+import { AppModel } from '../app/model'
+
+import Icon from './icon.svg'
+import { VisionModel } from './model'
+import { VisionNetwork } from './network'
+import { VisionView } from './view'
+
+export function installVision2({
+  nav,
+  appModel,
+  nusightNetwork,
+  Menu,
+}: {
+  nav: NavigationConfiguration
+  appModel: AppModel
+  nusightNetwork: NUsightNetwork
+  Menu: ComponentType
+}) {
+  const model = VisionModel.of(appModel)
+  nav.addRoute({
+    path: '/vision2',
```

Comment:
When we merge this PR, do we just want to replace vision1 with it?

Reply:
Yep! Once this is ready I'll raise a PR on top of this doing the rename/replacement, then merge them both together into master.